### DISCUSSION TITLE
Recovering Account

### DISCUSSION THREAD

**Discussion by metasoul111** [Web3Auth/Web3Auth#1151] [Help]
We are using Web2Auth in our DEX as part of the our log in process. If the user looses access to their email address attached to their Torus wallet we understand the email address cannot be changed and the backup phase has to be used to recover the account. Can this be done from the standard log in UI or do we need to migrate over to CustomAuth to provide this feature to our end users. Account recovery is critical to end users.

**Comment by Saviour1001** [comment]
You can do this using standard login UI, whichever `Web3Auth` SDK you are using, you can enable the [Multifactor Authentication (MFA)](https://web3auth.io/docs/sdk/web/modal/multi-factor-authentication) to allow users to create a backup share, MFA feature comes up with 4 values `default`, `optional`, `mandatory` and `none`, you can read in more detail regarding this in the link and choose as per your use case. 

Custom Authentication is a feature which allows you to authenticate users with your own custom authentication service.

**Reply by metasoul111** [reply]
@Saviour1001 thanks for the response, to clarify, how does the recovery process work? If the user has created an account with an email address and looses all access to the email address and then used the backup phrase to recover the account can they change the email at the time or how would they continue to access the account? Thanks.
